frontend/apps/marketing/src/components/carousels/actionBlockCarousel/ActionBlockCarousel.tsx

```diff
@@ -0,0 +1,133 @@
+'use client';
+
+import {BaseEntry, EntryFields} from 'contentful';
+import '@code-dot-org/component-library/carousel/index.css';
+import React, {ReactNode, useMemo} from 'react';
+
+import ActionBlock, {
+  ActionBlockProps,
+} from '@code-dot-org/component-library/actionBlock';
+import DSCOCarousel from '@code-dot-org/component-library/carousel';
+
+export type ActionBlockCarouselProps = ActionBlockProps & {
+  /** Carousel content w/ fields from Contentful */
+  slides: {
+    id: EntryFields.Text;
```

Comment:
nit: entries don't have this field so it can be removed

Reply:
Does this just need to be `string`?